projects/RPi/devices/Slice/patches/kodi/kodi-004-keyboard.patch

```diff
@@ -0,0 +1,22 @@
+--- a/system/keymaps/keyboard.xml	2016-09-17 16:35:20.000000000 +0100
++++ b/system/keymaps/keyboard.xml	2016-10-01 19:31:07.928719606 +0100
+@@ -56,7 +56,7 @@
+       <menu mod="longpress">Menu</menu>
+       <c>ContextMenu</c>
+       <c mod="longpress">Menu</c>
+-      <space>Pause</space>
++      <space>PlayPause</space>
+       <x>Stop</x>
+       <period>SkipNext</period>
+       <comma>SkipPrevious</comma>
+@@ -321,8 +321,8 @@
+       <l>NextSubtitle</l>
+       <left>StepBack</left>
+       <right>StepForward</right>
+-      <up>ChapterOrBigStepForward</up>
```

Comment:
I'm not absolutely sure about this, in general we always removed the volume capability because we expect the default to be 100% and allow the external amplifier / TV to control volume.  Otherwise all that is happening is the signal to noise ratio is being reduced...

I understand that having volume up/down is nice for analogue output, but it is being connected to an amp...